README.md

```diff
@@ -133,6 +133,10 @@ Note that pure components **will not report errors and will instead trap**, so t
 Note that features explicitly imported by the target world cannot be disabled - if you target a component to a world
 that imports `wasi:clocks`, then `disableFeatures: ['clocks']` will not be supported.
 
+## Using StarlingMonkey's `fetch-event`
+
+The StarlingMonkey engine provides the ability to use `fetchEvent` to handle calls to `wasi:http/incoming-handler@0.2.0#handle`. When targeting worlds that export wasi:http/incoming-handker@0.2.0 the fetch event will automatically be attached. Alternatively, to override the fetch event with a custom handler, export an explict incomingHandler or wasi:http/incoming-handler@0.2.0 object. Using the `fetchEvent` requires enabling the `http` feature. 
```

Comment:
Minor adjustments:

```suggestion
The StarlingMonkey engine provides the ability to use `fetchEvent` to handle calls to `wasi:http/incoming-handler@0.2.0#handle`. When targeting worlds that export `wasi:http/incoming-handler@0.2.0` the fetch event will automatically be attached. Alternatively, to override the fetch event with a custom handler, export an explict `incomingHandler` or `'wasi:http/incoming-handler@0.2.0'` object. Using the `fetchEvent` requires enabling the `http` feature. 
```

I'm just wondering about the `http` feature requirement. I suppose we will disable this once fetch is stable, but even then we need to think about what to do when `http` is disabled.

We should probably make it an error when http is disabled and the user hasn't exported the handler. Either just the error already at https://github.com/bytecodealliance/ComponentizeJS/pull/122/files#diff-69c4ddcc3664701394605dc159a10360361dc21afeee1a01c967c1f06f1f967aR398, or a new error that specifies the feature must be enabled or exported.

Reply:
I will pipe it down to the `bindgen` and create a new error.

Reply:
updated.